src/app/shared/pricing-definitions/pricing-definition/pricing-definition.component.html

```diff
@@ -252,11 +252,137 @@
                 </div>
               </div>
             </div>
+            <!-- form Restrictions -->
+            <div class="card">
```

Comment:
I can enter min duration which is greater than max duration: no error message is shown
Same for Min Energy.
When day of week is selected, the text should be white (poor contrast with black)

<img width="1158" alt="2022-01-24_17-48-53" src="https://user-images.githubusercontent.com/25266945/150827201-240e6722-d2a9-4a47-8e8f-f0f7a792ef20.png">